clang/lib/Serialization/ASTWriter.cpp

```diff
@@ -192,13 +201,13 @@ GetAffectingModuleMaps(const Preprocessor &PP, Module *RootModule) {
 
       // The containing module map is affecting, because it's being pointed
       // into by Module::DefinitionLoc.
-      if (auto FE = MM.getContainingModuleMapFile(Mod))
-        ModuleMaps.insert(*FE);
+      if (auto F = MM.getContainingModuleMapFileID(Mod); F.isValid())
+        ModuleMaps.insert(F);
       // For inferred modules, the module map that allowed inferring is not
       // related to the virtual containing module map file. It did affect the
       // compilation, though.
-      if (auto FE = MM.getModuleMapFileForUniquing(Mod))
-        ModuleMaps.insert(*FE);
+      if (auto F = MM.getModuleMapFileIDForUniquing(Mod); F.isValid())
```

Comment:
The duplicated name may not smell good.

Reply:
I am not sure I got the comment right, so wanted to ask again.

Do you mean the use of `F` for both `ifs`? The potential fix would be to use two different names (e.g. `F` and `UniquingF`).
Or is it this about something else (e.g. the logic here or shadowing of some names that aren't obvious to me).

Reply:
> Do you mean the use of F for both ifs? The potential fix would be to use two different names (e.g. F and UniquingF).

yes

Reply:
I've updated the second occurrence to use a different name (went with `UniqF` to avoid line breaks).
The same name did not bother me because it's scoped to just 2 lines, but I was still happy to update if you feel it's better.